CODE_OF_CONDUCT.md

```
@@ -0,0 +1,85 @@
+
+This Code of Conduct applies within all project spaces, and it also applies when
+an individual is representing the project or its community in public spaces.
+Examples of representing a project or community include using an official
+project e-mail address, posting via an official social media account, or acting
+as an appointed representative at an online or offline event. Representation of
+a project may be further defined and clarified by project maintainers.
+
+## Enforcement
+
+Instances of abusive, harassing, or otherwise unacceptable behavior may be
+reported by contacting the project team at yam@yam.dev or on Discord. All
```

markdownlint-cli2 (0.17.2)

Bare URL used

(MD034, no-bare-urls)